1. Продакт на главной mail.ru решил протестировать в рекомендательной ленте контента вместо карточек со статьями видеоплеер с короткими видео. Нынешний таймспент на юзера в день в среднем равен 25 минут, а стандартная ошибка (SD) равна 156. Мы предполагаем, что в новой версии таймспент на юзера в день изменится на 10%. Средний трафик 20000 человек в день. Посчитайте сколько дней необходимо держать эксперимент при alpha = 5% и beta = 20% .

Sample Size Calculation Results
Results and Live Interpretation
Assuming a pooled standard deviation of 156 units, the study would require a sample size of:
61124
for each group (i.e. a total sample size of 122248, assuming equal group sizes), to achieve a power of 80%
and a level of significance of 5% (two sided), for detecting a true difference in means between the test and
the reference group of 2.5 units.
In other words, if you select a random sample of 61124 from each population, and determine that the
difference in the two means is 2.5 units, and the pooled standard deviation is 156 units, you would have
80% power to declare that the two groups have significantly different means, i.e. a two sided p-value of less
than 0.05.
Reference: Dhand, N. K., & Khatkar, M. S. (2014). Statulator: An online statistical calculator. Sample Size
Calculator for Comparing Two Independent Means. Accessed 16 June 2023 at
http://statulator.com/SampleSize/ss2M.html
Note: Statulator used the input values of a power of 80%, a two sided level of significance of 5% and equal
group sizes for sample size calculation and adjusted the sample size for t-distribution. You may change the
options by clicking here or the 'Options' button and the adjustments by clicking here or the 'Adjust' button.


In [1]:
122248 / 20000

6.1124

Ответ: 6.1124 дней

2. Наша продуктовая команда в ecommerce магазине планирует запустить тест, направленный на ускорение загрузки сайта. Одна из основных метрик bounce rate в GA = 40%. Мы предполагаем, что при оптимизации сайта она изменится минимум на 20%.Средний трафик 4000 человек в день. Посчитайте сколько нам нужно дней держать эксперимент при alpha = 5% и beta = 20%

Sample Size Calculation Results
Results and Live Interpretation
The study would require a sample size of:
199
(number of pairs) to achieve a power of 80% and a level of significance of 5% (two sided), for detecting an
effect size of 0.2 between pairs .
In other words, if you select a random sample of 199 pairs and determine that the effect size is 0.2, you
would have 80% power to declare that the mean of the paired differences is significantly different from zero,
i.e. a two sided p-value is less than 0.05.
Reference: Dhand, N. K., & Khatkar, M. S. (2014). Statulator: An online statistical calculator. Sample Size
Calculator for Comparing Two Paired Means. Accessed 16 June 2023 at
http://statulator.com/SampleSize/ss2PM.html
Note: Statulator used the input values of a power of 80% and a two sided level of significance of 5% for
sample size calculation and adjusted the sample size for t-distribution. You may change the options by
clicking here or the 'Options' button and the adjustments by clicking here or the 'Adjust' button.

In [2]:
199 * 2 / 4000

0.0995

In [ ]:
Ответ: 0.0995 дней